build.gradle.kts

```diff
@@ -37,9 +37,9 @@ kotlin {
             )
 
             dependencies {
-                implementation("io.kotest:kotest-framework-engine:6.0.0.M1")
-                implementation("io.kotest:kotest-framework-api:6.0.0.M1")
-                implementation("io.kotest:kotest-assertions-core:6.0.0.M1")
+                implementation(libs.kotest.framework.engine)
+                implementation(libs.kotest.framework.api)
+                implementation(libs.kotest.assertions.core)
                 implementation("org.jetbrains:annotations:26.0.2")
```

Comment:
We probably should use the catalog for all our dependencies, which can be done as a follow-up. For now, I'm addressing the problems related to kotest.

Reply:
There is a Gradle plugin that could help make the migration to version catalog faster 

https://github.com/littlerobots/version-catalog-update-plugin

But I have not tried it so not sure if it works okay with KMP projects

Reply:
I think that for a small project with little dependencies, it even has some value to go one by one to understand what is used and how, especially grouped dependencies. Thanks for sharing, though!